app/assets/javascripts/darkswarm/directives/body_scroll.js.coffee

```diff
@@ -4,6 +4,12 @@ Darkswarm.directive "bodyScroll", ($rootScope, BodyScroll) ->
   link: (scope, elem, attrs) ->
     $rootScope.$on "toggleBodyScroll", ->
       if BodyScroll.disabled
-        elem.addClass "disable-scroll"
+        document.body.style.top = "-#{window.scrollY}px"
+        document.body.style.position = 'fixed'
+        document.body.style.overflowY = 'scroll'
       else
-        elem.removeClass "disable-scroll"
+        scrollY = parseInt(document.body.style.top)
```

Comment:
The `* -1` should probably be in this line because there is no negative scroll.

Reply:
this has been addressed in last commit